boot/boot.go

```diff
@@ -287,15 +287,13 @@ type BootableSet struct {
 	UnpackedGadgetDir string
 }
 
-// MakeBootable sets up the image filesystem with the given rootdir
-// such that it can be booted. This entails:
+// makeBootableUc16Uc18 setups the image filesystem for boot with UC16
+// and UC18 models. This entails:
 //  - installing the bootloader configuration from the gadget
 //  - creating symlinks for boot snaps from seed to the runtime blob dir
 //  - setting boot env vars pointing to the revisions of the boot snaps to use
 //  - extracting kernel assets as needed by the bootloader
-func MakeBootable(model *asserts.Model, rootdir string, bootWith *BootableSet) error {
-	opts := &bootloader.Options{PrepareImageTime: true}
-
+func makeBootableUc16Uc18(model *asserts.Model, rootdir string, bootWith *BootableSet, opts *bootloader.Options) error {
```

Comment:
the convention I used elsewhere is just to use 16 (to mean both 16/18, but I know we have situations where they are different),  and 20.  If we want the the full Uc in the suffix, then I think it should be UC16_18